tests/sdk_test/sdk_test.cpp

```diff
@@ -32,6 +32,8 @@ int main(int argc, char **argv)
     // Tests that requires a groupchat (start with public chat, converted into private)
     EXECUTE_TEST(t.TEST_PublicChatManagement(0, 1), "TEST Publicchat management");
     EXECUTE_TEST(t.TEST_GroupChatManagement(0, 1), "TEST Groupchat management");
+    // TODO: uncomment this test once the retention history support is deployed into shards 0 and 1 (currently, it only works in shard 2)
```

Comment:
Is it already in production?

Reply:
First we need to be able to set retention time to a few seconds (only for testing purposes) if we want to use this test.

Reply:
Yes